src/fmu/tools/sensitivities/_designsummary.py

```diff
@@ -34,7 +40,20 @@ def summarize_design(filename, sheetname="DesignSheet01"):
 
     """
 
-    # Initialisation of dataframe to store results
+    # Read design matrix
+    if str(filename).endswith(".xlsx"):
+        dgn = pd.read_excel(filename, sheetname, engine="openpyxl")
+        dgn.dropna(axis=0, how="all", inplace=True)
```

Comment:
Hm, this drops rows (realization) if a parameter entry is missing. Shouldn't we drop the column (the parameter) instead? Not sure just asking ....

Reply:
I notice that I've deleted the original comment that explains this line.
I think it makes sense to drop the row.